### DISCUSSION TITLE
Documentation suggestions

### DISCUSSION THREAD

**Discussion by tmlmt** [zauberzeug/nicegui#3155] [Ideas / Feature Requests]
Hi guys, thanks for the really good package! 👍

I've just completed an app for my personal use using nicegui, after having seen the limitations of streamlit. The experience has been  very good overall, but the learning curve has been steep and I wish I had read a few details in the documentation to achieve what I wanted:
- [x] how to run the app on https, either via nicegui itself or using nginx
- [x] combining ui.input with ui.time, or ui.date including:
    - preventing the time component from opening when clicking on the input field
    - and auto-closing once a date has been selected 
- [ ] responsiveness (or lack of) of plotly charts, which is by default in ui.plotly especially regarding titles
- [ ] how to make a ui.table responsive by using the `dense` or `grid` prop of quasar
- [ ] what I should expect in terms of reactivity/binding: here it would have been beneficial to see an explanation about the fact that only scalar variables are bindable and not objects. But that the ex4nicegui extension package exists

I would like to volunteer to add these to the docs. Do you confirm what these points qualify to be added and do you agree that I go ahead in proposing a PR?

**Comment by falkoschindler** [comment]
Thanks for the suggestions @tmlmt!
I think they would make a great addition to the documentation. We're looking forward to your pull request! 🙂

**Comment by vickorian** [comment]
@tmlmt 

I found an https example that's been working for me on reddit that was posted like 7 months ago. i'm dropping it here as it's pretty straight forward and if you're adding it you can choose which method you find easier in the documentation to implement. [The link is here.](https://www.reddit.com/r/nicegui/comments/13xe1aa/how_to_use_https_instead_of_http/)
